lib/rails_admin.rb

```diff
@@ -33,6 +34,28 @@ def self.config(entity = nil, &block)
       RailsAdmin::Config
     end
   end
+
+  # Backwards-compatible with safe_yaml/load when SafeYAML isn't available.
+  # Evaluates available YAML loaders at boot and creates appropriate method,
+  # so no conditionals are required at runtime.
+  begin
+    require 'safe_yaml/load'
+    def self.yaml_load(yaml)
+      SafeYAML.load(yaml)
+    end
```

Comment:
define `self.yaml_load` to use SafeYAML in non-monkey-patching mode, when available